### PR TITLE
Add "When I turn off [mode]"

### LINKED ISSUE #38: How to turn off a mode?
I'm new to a lot of this, so my guess based on is that in emacs:

- `C-u M-x my-mode` unconditionally enables the mode
- `C-u -1 M-x my-mode` unconditionally disables the mode?

I'm trying to define a helper to turn off the mode, but I don't know how to add the universal arguments to the vector: `[?\C-u \- 1 ?\M-x]` does not seem to work, and I don't quite know what changes I need to make.

I was also surprised that this didn't exist as a step in espuds :)

### PR DESCRIPTION
**Pull request by davidshepherd7**
Fixes #38.

Force travis to run the tests for me, since they don't quite work on emacs 25

### CODE REVIEW

File: test/espuds-test.el [hunk 1]
```diff
@@ -413,6 +413,15 @@
     (When "I turn on abbrev-mode"))
    (should abbrev-mode)))
 
+(ert-deftest when-i-turn-on-and-off-minor-mode ()
+  "Should turn off minor mode."
+  (with-playground
+   (with-mock
+    (stub message)
+    (When "I turn on abbrev-mode")
+    (When "I turn off minor mode abbrev-mode"))
```
Comment (davidshepherd7) [inline test/espuds-test.el]: I'm not a fan of this asymmetry between turning on and off, but I think it's important to make it clear that you can only turn off minor modes.

Comment (rejeep) [inline test/espuds-test.el]: I don't mind, I think it's better actually, since it's more clear.

File: test/espuds-test.el [hunk 2]
```diff
@@ -413,6 +413,15 @@
...
+   (should (not abbrev-mode))))
```
Comment (rejeep) [inline test/espuds-test.el]: Use `should-not`

File: espuds.el [hunk 3]
```diff
@@ -326,6 +326,12 @@ chain. Otherwise simulate the TYPING."
     (let ((v (vconcat [?\C-u ?\M-x] (string-to-vector mode))))
       (execute-kbd-macro v))))
 
+(When "^I turn off minor mode \\(.+\\)$"
+  "Turns off some minor mode (to turn off a major mode you have to turn on another one)."
```
Comment (rejeep) [inline espuds.el]: Can you please put the text in params on a separate line (just like how defuns should be documented).

**Comment by davidshepherd7** [comment]
Fixed :+1:

**Comment by rejeep** [comment]
Thanks! 👍

**Pull request merged**